src/Controllers/OAuthController.php

```diff
@@ -75,13 +75,19 @@ public function callBack(): RedirectResponse
 
         $find = ['email' => $userInfo->email];
 
-        if ($this->checkExistenceUser($find) === true) {
-            $updateFildes = $oauthClass->getColumnsName('syncingUserInfo', $userInfo);
+        if ($this->checkExistenceUser($find)) {
+            $updateFields = $oauthClass->getColumnsName('syncingUserInfo', $userInfo);
```

Comment:
Good catch . In order to have a better contribution, you should provide an explanation related to the change you made in the commit message. This will improve the visibility and maintenance of the code.
[Here ](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#committing)is a reference for how to write a message for commits. It is better to follow these principles.

Reply:
noted, thanks for advance